metricbeat/tests/system/test_haproxy.py

```diff
@@ -10,17 +10,7 @@ class Test(metricbeat.BaseTest):
 
     COMPOSE_SERVICES = ['haproxy']
 
-    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
-    def test_info(self):
-        """
-        haproxy info metricset test
-        """
-        self.render_config_template(modules=[{
-            "name": "haproxy",
-            "metricsets": ["info"],
-            "hosts": self.get_hosts(),
-            "period": "5s"
-        }])
+    def _test_info(self):
```

Comment:
Why did you add an `_` in front and remove the intergration part line?

Reply:
I moved common code to "internal" functions so it can be reused for multiple cases. At the end this test has only the tcp socket case (it could maybe have also another case for a unix socket file), but the stat test has multiple cases.

Reply:
Got it. The diff tricked me here :-)